README.md

```diff
@@ -10,15 +10,31 @@ dependencies, such as Node, PostgreSQL, ElasticMQ, pgcli, etc.
 
 [Official documentation](https://nixos.org/manual/nix/stable/#sect-macos-installation)
 
-### Step-by-step (tested on Big Sur)
-
-This has been tested on a new install of macOS 12 (Monterey)
+### For Intel architecture
 
 ```bash
 # Install Apple command line developer tools
 xcode-select --install
 
-# If you're on an M1 mac, install rosetta
+# Install nix itself in multi-user mode
+sh <(curl -L https://nixos.org/nix/install) --daemon
+
+# Start a new terminal session to ensure `nix` is in your path
+which nix # to confirm you have `nix` in your path
+
+# (For M1) Create nix configuration to enable Intel platform
```

Comment:
i'm confused about the "For M1" piece of this comment

Reply:
Oops, yeah, copy paste fail. Will fix!